README.md

```diff
@@ -270,6 +272,47 @@ To run the tests:
 bundle exec rspec
 ```
 
+### Docker
+
+A Dockerfile is provided in order to interact with the application in development.
+
+Build the docker image:
+
+```sh
+docker-compose build app
+```
+
+Bring up the docker container and it's dependencies:
+
+```sh
+docker-compose up -d
+```
+
+Initialize the database:
+
+```sh
+docker-compose run app bundle exec rails db:reset
```

Comment:
This can be taken down to one line too, since you can pass multiple tasks to `rake`/`rails`, which will be executed in order: `docker-compose run app bundle exec rails db:reset db:seed`